src/main/java/tc/oc/pgm/api/setting/SettingValue.java

```diff
@@ -17,11 +17,17 @@
   CHAT_TEAM("chat", "team"), // Only send to members on the player's team
   CHAT_GLOBAL("chat", "global"), // Send to all players in the same match
   CHAT_ADMIN("chat", "admin"), // Send to all server operators
+
   DEATH_OWN("death", "own"), // Only send death messages involving self
   DEATH_ALL("death", "all"), // Send all death messages, highlight your own
+
   PICKER_AUTO("picker", "auto"), // Display after cycle, or with permissions.
   PICKER_ON("picker", "on"), // Display the picker GUI always
-  PICKER_OFF("picker", "off"); // Never display the picker GUI
+  PICKER_OFF("picker", "off"), // Never display the picker GUI
+
+  JOIN_ON("join", "all"), // Send all join messages
+  JOIN_OFF("join", "none"), // Never send join messages
```

Comment:
```suggestion
  JOIN_OFF("join", "none") // Never send join messages
```

Reply:
Why this change?

Reply:
Doesn't make sense to leave a trailing comma at the end

Reply:
If somebody adds more settings later, less lines are going to be affected. Makes resolving conflicts easier.